torch/_refs/__init__.py

```diff
@@ -3262,6 +3261,7 @@ def _reshape_view_helper(a: TensorLikeType, *shape, allow_copy: bool) -> TensorL
 # NOTE: shape is a vararg because Tensor.reshape can be called with as
 # Tensor.reshape(a, b, c) or Tensor.reshape((a, b, c)) Function call
 # torch.reshape doesn't support unpacked shapes
+@aten.reshape.default.py_impl(DispatchKey.CompositeImplicitAutograd)
```

Comment:
This appears to be deeply problematic. I'll probably figure out another way to do this.